server/db/config/connection.js

```diff
@@ -0,0 +1,6 @@
+const { Pool } = require('pg')
+
+module.exports = new Pool({
+  connectionString: process.env.DATABASE_URL,
```

Comment:
good job here but you need to handle the error
what if there is no  DATABASE_URL this will break our server!!!
so handle the error here

Reply:
Done